src/shutdown/shutdown.c

```diff
@@ -400,6 +400,9 @@ int main(int argc, char *argv[]) {
         /* Lock us into memory */
         (void) mlockall(MCL_CURRENT|MCL_FUTURE);
 
+        if (mount(NULL, "/", NULL, MS_REC|MS_PRIVATE, NULL) < 0)
+                log_warning_errno(errno, "Failed to make mounts private: %m");
```

Comment:
we try to suffix log messages about errors we ignore with `, ignoring`. i.e. maybe change this to:

```
log_warning_errno(errno, "Failed to make mounts private, ignoring: %m");
```